README.md

```diff
@@ -133,6 +136,25 @@ Each item in `phases` must be an object containing:
 If no effective date is provided, then the phase takes effect
 immediately.
 
+If `info` is provided, it updates the org with info in the same
+way as calling `updateOrg(org, info)`.
+
+### `updateOrg(org, info)`
```

Comment:
The convenience it adds is not having to pass 2 undefined args, like `tier.subscribe('org:o', undefined, undefined, orgInfo)`, if you do want to update the data later on without adding a new subscription.  Also, then it's clear that all you're doing is updating the org info; it's a bit weird to call `tier.subcribe()` to do something other than creating a subscription.  Makes perfect sense internally that they're the same endpoint, and being _able_ to set that in the subscribe()/schedule() call is reasonable, since that's also when we might be creating the organization initially, of course.

Reply:
To keep as close to the reference implementation as possible, use `Subscribe`, and `Schedule`, where `Schedule` takes the `OrgInfo`, optionally, which is consistent with our recent discussions where we agreed that when we speak of using phases to describe the "schedule" we're "scheduling" and when we're speaking of immediate changes that go on indefinitely we say "subscribe".

Please see our official reference implementation in Go:

https://github.com/tierrun/tier/blob/292b224c79fd606be6919f83faadb8e486b2718c/client/tier/client.go#L184
https://github.com/tierrun/tier/blob/292b224c79fd606be6919f83faadb8e486b2718c/client/tier/client.go#L200

Keeping names and terms consistent with the reference SDK helps with communicating with users of Tier no matter which language they are working in.

On that note, It would be nice to see `limits` have an alias `lookupLimits`, as with the others that diverged, and `limits` be deprecated. But that is for another PR.